txwrapplog.py

```diff
@@ -22,8 +23,10 @@
 }
 
 
-def wrapp_observer(output):
+def wrapp_observer(output, service=None, host=None):
     json = jsonFileLogObserver(output, recordSeparator='')
+    service = service or os.environ.get('SERVICE_NAME')
```

Comment:
I don't think that would work for two reasons:

- I don't think it would be good if i forcefully ask the user to provide a service and host.
- This strategy would not be backward compatible if we needed to change it later
- If i forcefully allow the function to accept a service, there is no point in falling back to the env variable.

And lastly, what's really wrong with optionally asking for the service name?